api/base.py

```diff
@@ -182,9 +180,15 @@ def set_origin(self, drone_request, drone_name):
                 'id': self.uid
             }
         elif drone_request:
+
+            method = self.request.headers.get('X-SciTran-Method')
+            name = self.request.headers.get('X-SciTran-Name')
```

Comment:
Propose that we add `.strip()` to the end of each of these to remove whitespace.